packages/playground/config.json

```diff
@@ -1,4 +1,5 @@
 {
+  "configFile": "./sentry.properties",
```

Comment:
Is this still needed? As far as I can see we don't have a properties file

Reply:
Nope :D